lib/addic7ed/subtitle.rb

```diff
@@ -53,10 +55,22 @@ def normalize_version!
       @version.upcase!
     end
 
+    def normalize_comment!
+      @comment ||= ""
+      @comment.downcase!
+    end
+
     def is_compatible_with?(other_version)
-      self.version == other_version ||
+      version_compatible = self.version == other_version ||
       COMPATIBILITY_720P[self.version] == other_version ||
       COMPATIBILITY_720P[other_version] == self.version
+
+      comment_compatible = self.comment.include? other_version.downcase
+      if COMPATIBILITY_720P[other_version]
+        comment_compatible ||= self.comment.include? COMPATIBILITY_720P[other_version].downcase
+      end
+
+      version_compatible || comment_compatible
```

Comment:
This looks like a sign that we should split ths method in 2 methods :smiley: 
Something like `generally_compatible?` and `commented_as_compatible?`, maybe ?